bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
+
+            # need to add tool to all underlying plots
             for plot in self._plots:
+                # only add tools if the underlying plot hasn't been customized
+                # by some user injection
                 if not plot.tools:
-                    if not self.categorical:
-                        pan = PanTool()
-                        wheelzoom = WheelZoomTool()
-                        reset = ResetTool()
-                        plot.add_tools(pan, wheelzoom, reset)
-                    previewsave = PreviewSaveTool()
-                    plot.add_tools(previewsave)
+                    tools = []
+                    # if True let's create the default tools
+                    if isinstance(self.tools, bool) and self.tools:
+                        if not self.categorical:
+                            tools = [PanTool(),  WheelZoomTool(), ResetTool()]
+                        tools.append(PreviewSaveTool())
```

Comment:
yes, btw pan is already included in not categorical ones... I guess we have to follow the default ones here: https://github.com/bokeh/bokeh/blob/feature/configurable_charts_tools/bokeh/plotting.py#L44